common/app/common/NewNavigation.scala

```diff
@@ -1,6 +1,8 @@
 package common
 
-case class NavLink(name: String, url: String)
+import conf.Configuration
+
+case class NavLink(name: String, url: String, iconName: String = "")
```

Comment:
That bit is just defined that `iconName` is a string and that it's default is an empty string. The only place it is being used is in the `MembershipLinks object`. None of the other `NavLink`s have it so that is why I put a default